uniswap_swap_bot/src/bot.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+const handleTransaction: HandleTransaction = async (
+  txEvent: TransactionEvent
+) => {
+  const findings: Finding[] = [];
+
+  const tokenSwapEvent = txEvent.filterLog(SWAP_EVENT, UNISWAP_ADDRESS);
```

Comment:
While you're on the right path, what we're looking for is for a `swap` that happens on _any valid_ UniswapV3 pool. Currently, you are filtering for instances where the swap is happening on the UniswapV3 Factory at [0x1F98431c8aD98523631AE4a59f267346ea31F984](https://etherscan.io/address/0x1F98431c8aD98523631AE4a59f267346ea31F984).

So the logic needs to be updated so that any `swap` across the numerous UniswapV3 pools gets alerted.